README.md

```diff
@@ -1,3 +1,74 @@
-# slashml-python-client
+# SLASHML Python client
 
-to be updated to guide users
+This is a Python client for SLASHML. It lets you run transcription jobs from your Python code or Jupyter notebook. Do a transcription job with three lines of code
```

Comment:
add url for slashml

Reply:
The readme should be generic, so it doesn't run transcrption jobs. It runs machine learning jobs.

Reply:
Then you can give an example of how to do transcription as an example.

Reply:
done